Loading dataset count: 29
Skipped videos 73/27764=0.262930%
max text len 0
Loading dataset count: 4
Skipped videos 22/3964=0.554995%
max text len 0
Starting epoch 0 out of 100
torch.Size([50, 1485])
x torch.Size([74, 50, 512])
y gp output torch.Size([50, 28])
Traceback (most recent call last):
  File "gpt2train.py", line 25, in <module>
    main(args)
  File "gpt2train.py", line 10, in main
    target_device=torch.device("cuda"), cache_in_ram=False, temporal_aug=args.temporal_aug)
  File "/scratch/mll469/gpt2deeplr/combined/gpt2_train.py", line 47, in run
    val_losses, train_cers, val_cers, train_wers, val_wers)
  File "/scratch/mll469/gpt2deeplr/combined/gpt2_train.py", line 160, in train
    logits = model(images_tensor, ranked_predictions)
  File "/scratch/mll469/env_train/lib/python3.7/site-packages/torch/nn/modules/module.py", line 541, in __call__
    result = self.forward(*input, **kwargs)
  File "/scratch/mll469/gpt2deeplr/combined/gpt2_model.py", line 85, in forward
    xy = torch.cat((x, y))
RuntimeError: invalid argument 0: Tensors must have same number of dimensions: got 2 and 3 at /pytorch/aten/src/THC/generic/THCTensorMath.cu:62
